Creation of FV3 diag table documentation (#2277)
ufs-community/ufs-weather-model: doc/UsersGuide/source/InputsOutputs.rst

Comments can be added to the diag_table using the hash symbol (``#``).

> .. list-table:: *Each WM component has its own diag_table with associated varialbes. Below is the full set of options.*

Reviewer: ```suggestion

Each WM component has its own ``diag_table`` with associated variables. :numref:`Table %s <diag-table-options>` contains links to the full set of options for each WM component.

.. _diag-table-options:

.. list-table:: * *diag_table* Options for WM Components*
```

Comments can be added to the diag_table using the hash symbol (``#``).

Each WM component has its own ``diag_table`` with associated variables. :numref:`Table %s <diag-table-options>` contains links to the full set of options for each WM component.

.. _diag-table-options:

.. list-table:: * *diag_table* Options for WM Components*
   :widths: 16 24 
   :header-rows: 1

>    * - WM Component
>      - Diag Table
>    * - FV3
>      - :ref:`FV3 Variables <fv3diagtable>`
>    * - MOM6
>      - :ref:`MOM6 Variables <mom6diagtable>`

Reviewer: ```suggestion
   * - WM Component
     - Diag Table
     - Source File
   * - FV3
     - :ref:`FV3 Variables <fv3diagtable>`
     - `GFS_diagnostics.F90 <https://github.com/NOAA-EMC/fv3atm/blob/develop/ccpp/driver/GFS_diagnostics.F90>`_
   * - MOM6
     - :ref:`MOM6 Variables <mom6diagtable>`
     - `MOM_diagnostics.F90 <https://github.com/NOAA-EMC/MOM6/blob/main/src/diagnostics/MOM_diagnostics.F90>`_
```
Just added Source File field to table so that people can go look at the original source code if they want.

Comments can be added to the diag_table using the hash symbol (``#``).

Each WM component has its own ``diag_table`` with associated variables. :numref:`Table %s <diag-table-options>` contains links to the full set of options for each WM component.

.. _diag-table-options:

.. list-table:: * *diag_table* Options for WM Components*

>    :widths: 16 24 

Reviewer: ```suggestion
   :widths: 16 24 24
```
Since there are 3 columns, we need three widths or the docs won't build the table.

   :header-rows: 1

   * - WM Component
     - Diag Table
     - Source File
   * - FV3
     - :ref:`FV3 Variables <fv3diagtable>`
     - `GFS_diagnostics.F90 <https://github.com/NOAA-EMC/fv3atm/blob/develop/ccpp/driver/GFS_diagnostics.F90>`_
   * - MOM6

>      - :ref:`MOM6 Variables <mom6diagtable>`

Reviewer: ```suggestion
     - `MOM6 Variables <https://ncar.github.io/MOM6/APIs/namespacemom__diagnostics.html>`_
```
We can link to the MOM6 documentation since you moved your in-progress MOM6 docs out.
